### ISSUE TITLE
Undefined symbol: _OBJC_CLASS_$_QBRTCs

### ISSUE THREAD

**Issue by jayroland88** [QuickBlox/quickblox-ios-sdk#1372]
**New Issue Checklist**

 - [ yes] I have read the [Documentation](https://quickblox.com/developers/IOS)
 - [yes] I have searched for a similar issue in the [project](https://github.com/QuickBlox/quickblox-ios-sdk/issues) and found none

**Environment details**

Info                           | Value                                       |
-------------------------------|---------------------------------------------|
iOS Version                    | 13.0                                    |                                          
Quickblox iOS SDK version      | e.g. 2.9.                                  |
QuickbloxWebRTC SDK version    | e.g. 2.8.1                                  |                               
Xcode Version                  |  Xcode 15.3                              |
Repro with our demo sample     | YES   |

**Did this work before?**
Yes it did, then i upgraded to 2.19

**Expected behavior**
no errors during building of project

**Actual behavior**
These errors appear:
![Screenshot 2024- ](https://github.com/QuickBlox/quickblox-ios-sdk/assets/168790058/701e7446-3b89-4bf8-903c-a4fda169a09a)
 
</details>

**Steps to reproduce the behavior**
1. Use M1 Laptop arm64: 
2. create xcode swift project use spm and add https://github.com/QuickBlox/ios-quickblox-sdk.git  & https://github.com/QuickBlox/ios-quickblox-sdk-webrtc.git 
3. try to access any of these and build and see error:
4. // QBRTCBaseSession
// QBRTCCameraCapture
// QBRTCClient
// QBRTCConfig
// QBRTCRemoteVideoView
// QBRTCSession
// QBRTCVideoFormat
//Undefined symbol: _OBJC_CLASS_$_QBRTCAudioSession
//Undefined symbol: _OBJC_CLASS_$_QBRTCAudioSessionConfiguration
//Undefined symbol: _OBJC_CLASS_$_QBRTCClient
//Undefined symbol: _OBJC_CLASS_$_QBRTCConfig
//Undefined symbol: _OBJC_CLASS_$_QBRTCClient

**Any others comments**
-N/A

**Comment by IlliaChemolosov** [comment]
@jayroland88 Hello there!

Could you please specify whether you're attempting to run the project using the simulator or a physical device when encountering these errors?

**Comment by jayroland88** [comment]
Simulator - should not work with  https://github.com/QuickBlox/ios-quickblox-sdk.git I get that.

But these:
// QBRTCClient
// QBRTCConfig
// QBRTCSession
should be in  https://github.com/QuickBlox/ios-quickblox-sdk.git correct?

**Comment by IlliaChemolosov** [comment]
@jayroland88  Hello

The QuickbloxWebRTC SDK currently does not support simulators, thus call-related samples may encounter issues when attempting to run on the simulator. We are actively developing a solution to enable simulator support in the upcoming version.

**Comment by mendimr** [comment]
> @jayroland88 Hello
> 
> The QuickbloxWebRTC SDK currently does not support simulators, thus call-related samples may encounter issues when attempting to run on the simulator. We are actively developing a solution to enable simulator support in the upcoming version.

I don't get it how and why this is not fixed, M1 was released in 2020, all devs are moving to it and still there is no support for it. This makes developing really hard since no SwiftUI preview or any UI testing can be done in the Simulator.

Our aim is not to test QBWebRTC in simulators just to be clear.
I tried running [WebRTC](https://github.com/stasel/WebRTC) and it builds perfectly on my simulator.
